middeware/errorHandler.js

```diff
@@ -0,0 +1,13 @@
+const  errorHandler = (e)=>{
```

Comment:
What is `e`? Would `fn` or `function` not be a better parameter name?

Reply:
ok sir